pkg/datapath/linux/linux_defaults/linux_defaults.go

```diff
@@ -23,6 +23,12 @@ const (
 	// RouteTableIPSec is the default table ID to use for IPSec routing rules
 	RouteTableIPSec = 200
 
+	// RouteTableInterfacesOffset is the offset for the per-ENI / Azure
+	// interface tables. Each ENI / Azure interface will have its own table
+	// starting with this offset. It is 10 because it is unlikely to collide
+	// with the main routing table which is between 253-255. See ip-route(8).
+	RouteTableInterfacesOffset = 10
```

Comment:
FYI: This is already used in previous commits.

Reply:
Thanks, fixed.

Reply:
Wouldn't this collide with the IPSec table (see above `RouteTableIPSec`)?

Reply:
I think it's OK as the maximum interface number is currently 15, so the maximum route table number will be 25, which is well short of the 200 required to conflict with `RouteTableIPSec`. It may well be worth updating the comment, and even adding a check in the code however.

Reply:
OK, so my question is it guaranteed that the ENI Number will always be in the [1;15] even after multiple create and delete?

Reply:
Yes

Reply:
Thanks. Yes that's worth adding. Will do